dist/src/main/resources/mc_locale/mc_locale_java_en_US.properties

```diff
@@ -56,6 +56,10 @@ MQTT=MQTT
 NEW=New
```

Comment:
@Fraid We always add changes in `mc_locale_java-source_en.properties`, not in `mc_locale_java_en_US.properties`. However, to test it locally you can change on your local files.